### ISSUE TITLE
Update UI for better accessibility

### ISSUE THREAD

**Issue by Dafondo** [unc-cs-sg/carolinacupboard-inventory#34]
**Context:**
1. We want people of all ability to be able to use the app.
1. There are a variety of accessibility violations when running [WAVE](https://wave.webaim.org/) or [axe](https://www.deque.com/axe/), which follow WCAG 2.0 and Section 508 guidelines.

**Consider:**
1. Running WAVE/axe, fixing all accessibility violations, and following suggested best practice.
1. Running manual accessibility tests with a colorblind simulator and screen reader.

**Done:**
1. No violations are shown when running WAVE/axe.
1. Ensured, to the best of our ability, that the app is accessible to all visitors.
